app/grandchallenge/reader_studies/utils.py

```diff
@@ -7,6 +9,20 @@
 from grandchallenge.reader_studies.models import Answer, DisplaySet
 from grandchallenge.workstations.models import Workstation
 
+logger = logging.getLogger(__name__)
+
+
+def _get_civ(image, slug):
+    try:
+        ci = ComponentInterface.objects.get(slug=slug)
+    except ComponentInterfaceValue.DoesNotExist:
+        raise ValueError(f"ComponentInterface {slug} does not exist.")
+
+    civ, _ = ComponentInterfaceValue.objects.get_or_create(
```

Comment:
This can be problematic in cases where there are multiple CIVs for the same interface and image, shouldn't happen but it is an issue in prod due to how things ran historically.

Reply:
So should we use a filter and create a new civ if one does not exist already?

Reply:
Yes, filter and take the first, if it doesn't exist create it.